dashboard/src/pages/MyChapters.vue

```diff
@@ -59,7 +59,7 @@ const scheduled_events = createListResource({
   fields: ['name', 'event_name', 'event_type', 'chapter_name', 'status', 'event_start_date'],
   filters: {
     event_end_date: ['>', new Date()],
-    status: ['in', ['', 'Draft', 'Approved', 'Live']],
+    status: ['in', ['', 'Draft', 'Live']],
```

Comment:
`''` isn't a valid option anymore, is it?

Reply:
Made changes